tests/rule_based_profiler/test_rule_based_profiler.py

```diff
@@ -1349,7 +1367,8 @@ def test_run_with_expectation_suite_arg(mock_data_context: mock.MagicMock):
     suite: ExpectationSuite = ExpectationSuite(
         expectation_suite_name="my_expectation_suite"
     )
-    result_suite: ExpectationSuite = profiler.run(expectation_suite=suite)
+    profiler.run()
+    result_suite: ExpectationSuite = profiler.expectation_suite(expectation_suite=suite)
```

Comment:
This is more of a "flow" concern, but would you consider a re-name of the method `expectation_suite()`? 

would love to get the thoughts of the rest of the team on this, but it doesn't feel so smooth to go from 

profiler.run() : "ok the profiler is doing something" (ie changing its "state")
`profiler.expectation_suite()` and then have to pass in a `suite_name` or `suite`. 


An option could be something like `get_expectation_suite()`, but it feels weird to have to pass in an empty suite in this case ( "don't we already have the suite?") Or `populate_expectation_suite()`, but that's a term we haven't used before.

Would love to get your thoughts on this @alexsherstinsky

Reply:
@Shinnnyshinshin good point -- I already renamed it like you suggested (independently!) in the upcoming PR -- for the same reasons.  Thanks!